Release 1.7.0: update changelog and shared to v170 (#227)
NordicSemiconductor/pc-nrfconnect-rssi: Changelog.md

## 1.7.0 - 2024-04-08

### Changed

> -   Moved feedback tab to a dialog which can be opened by going to the about tab
>     and click **Give Feedback**.

Reviewer: ```suggestion
-   Moved the Feedback tab to the **Give Feedback** dialog in the About tab.
```

Author: What about "Moved the Feedback tab into the **Give Feedback** dialog in the About tab." ?
make concise back again :D

Reviewer: I shortened my proposal to `` Integrated the functionality of the Feedback tab in the **Give Feedback** dialog in the About tab.``
The reason why I didn't like the "Moved..." proposal was that you didn't move the tab -- there's no tab under Give Feedback -- you just moved the functionality.
